modules/swagger-servlet/src/test/java/io/swagger/servlet/extensions/ApiOperationTypeTest.java

```diff
@@ -0,0 +1,21 @@
+package io.swagger.servlet.extensions;
+
+import io.swagger.models.Swagger;
+import io.swagger.servlet.Reader;
+import io.swagger.servlet.resources.ApiOperationResource;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+public class ApiOperationTypeTest {
+
+    private static final String USERS_PATH = "/users";
+
+    @Test
+    public void testApiOperationInType() {
+        final Swagger swagger = new Swagger();
+        Reader.read(swagger, Collections.<Class<?>>singleton(ApiOperationResource.class));
+        Assert.assertNotNull(swagger.getPaths().get(USERS_PATH));
```

Comment:
This doesn't really test the `ApiOperation` annotation..

Reply:
Renamed to testThatApiOperationMightBeInType.